src/fhirConfig.ts

```diff
@@ -151,11 +151,30 @@ export interface ProductInfo {
     copyright?: string;
 }
 
+export interface MultiTenancyConfig {
+    /**
+     * Enable multi-tenancy.
+     * This value is always true. To disable multi tenancy, remove the `multiTenancyConfig` altogether.
```

Comment:
I am confused how this works in deployment package-- if I didn't want MT I just don't pass in this object?

Maybe a better question is why have a static boolean

Reply:
> if I didn't want MT I just don't pass in this object?

correct

> why have a static boolean

yeah, the `enableMultiTenancy` field is a bit redundant, but enabling multi-tenancy is a big breaking change and I don't like for it to be implicitly enabled just by adding `MultiTenancyConfig`.